WalletWasabi/Services/UpdateChecker.cs

```diff
@@ -35,7 +35,7 @@ private set
 		public UpdateChecker(TimeSpan period, WasabiClient client) : base(period)
 		{
 			WasabiClient = Guard.NotNull(nameof(client), client);
-			UpdateStatus = new UpdateStatus(true, true);
+			UpdateStatus = new UpdateStatus(true, true, new Version(0, 0));
```

Comment:
There is a `Version.ctor` that takes no parameters and sets major and minor to zero by default.

Reply:
Done.